dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -145,9 +146,14 @@ def readRecord(self,reader,writer,file_type,interfaces,row_number,job_id,fields,
         reduce_row = False
         row_error_found = False
         try:
-
-            record = FieldCleaner.cleanRow(reader.get_next_record(), self.long_to_short_dict, fields)
+            (next_record, flex_cols) = reader.get_next_record()
+            record = FieldCleaner.cleanRow(next_record, self.long_to_short_dict, fields)
             record["row_number"] = row_number
+            if not flex_cols:
+                flex_cols = None
+            else:
+                flex_cols["row_number"] = row_number
+
             if reader.is_finished and len(record) < 2:
                 # This is the last line and is empty, don't record an error
                 return {}, True, True, True, False  # Don't count this row
```

Comment:
Sure thing! The other two `return`s in this function are returning a six-element tuple, but this one is only returning a five-element tuple. I think you just missed one `return` when you added that sixth element to the other two.